### PR TITLE
fix(docs): small grammar fixes & not correct words

### PR DESCRIPTION
**Pull request by daniloff200**
Bunch of small grammar fixes, which were found during the docs learning and researching

### CODE REVIEW

File: docs/reference/authorization.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ See [this example for authenticating with the AWS CLI](../using/aws_cli.md).
 
 Access to resources is managed very much like [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/intro-structure.html){:target="_blank"}.
 
-There are 4 basic components to the system:
+There are 4 basic components of the system:
```
Comment (arielshaqed) [inline docs/reference/authorization.md:50]: Please revert this change, "to" is correct here and is common.  In this sentence "of" would imply that there can be other components.

File: docs/reference/import.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Unfortunately, copying data is not always feasible for the following reasons:
 3. It requires you to switch to using the lakeFS endpoint in all places at once.
 
 ## Using lakeFS import tool
-To solve this, we offer an import tool which will not copy any data and allow for a more gradual onboarding process.
+To solve this, we offer an import tool that will not copy any data and allow for a more gradual onboarding process.
```
Comment (arielshaqed) [inline docs/reference/import.md:30]: Nice.  Mind trying this?
```suggestion
To solve this we offer an import tool that does not copy any data, allowing for a more gradual onboarding process.
```

**Review by arielshaqed — COMMENTED** [review]
Great, thanks!

Like any grammar fix PR I have 2 small requests, inline.

File: docs/reference/authorization.md (hunk 1 above)
Comment (daniloff200) [reply docs/reference/authorization.md]: done

File: docs/reference/import.md (hunk 2 above)
Comment (daniloff200) [reply docs/reference/import.md]: Yeah, I think we can use that.
Thanks for the suggestion!

**Review by arielshaqed — APPROVED** [review]
Great, thanks!  fixes a bug and more readable.

**Comment by arielshaqed** [comment]
Waiting for tests (which should anyway pass...) to pull.
